src/types/index.d.ts

```diff
@@ -10,7 +10,13 @@ export interface FluentVueOptions {
   warnMissing?: ((key: string) => void) | boolean
 
   /** Custom function for parsing markup */
-  parseMarkup?: (markup: string) => SimpleNode[]
+  parseMarkup?: (markup: string) => SimpleNode[],
+
+  /** Override the names of the global functions and directive to avoid conflicts */
+  globals?: {
+    t?: string;
+    ta?: string;
+  }
```

Comment:
I would prefer options to be like this:

```ts
globals?: {
  functions?: {
    format?: string
    formatAttrs?: string
  },
  component?: string
  directive?: string
}
```

So directive, component and functions could be configured independently.
And this will eliminate a need for concatenation with `$` in the code. Just make `globals.functions.format` be `$t`.

Reply:
That seems more logical! I've pushed these changes up.